src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -285,393 +301,251 @@ def generateBaseMesh(cls, region, options):
+                ( 153, [ [  -2.089,  -7.882,  10.275], [   0.726,   0.578,   0.576], [  -1.142,   1.396,   0.120], [   0.245,   0.671,   3.273] ] ),
+                ( 154, [ [  -1.356,  -7.061,  10.926], [   0.730,   1.056,   0.719], [  -1.311,   0.948,  -0.093], [  -0.439,  -0.402,   1.866] ] ),
+                ( 155, [ [  -0.671,  -5.759,  11.693], [   0.637,   1.539,   0.809], [  -1.332,   0.757,  -0.196], [  -1.294,  -1.686,   0.998] ] ),
+                ( 156, [ [  -3.199,  -6.469,  10.436], [   0.461,   0.435,   0.343], [  -1.083,   1.481,   0.161], [   0.359,   0.555,   2.294] ] ),
+                ( 157, [ [  -2.662,  -5.887,  10.861], [   0.614,   0.728,   0.505], [  -1.200,   1.378,   0.011], [  -0.303,  -0.255,   1.632] ] ),
+                ( 158, [ [  -1.982,  -5.004,  11.443], [   0.743,   1.037,   0.660], [  -1.291,   1.074,  -0.185], [  -0.992,  -1.135,   0.958] ] ),
+                ( 159, [ [  -4.250,  -4.921,  10.596], [   0.526,   0.538,   0.338], [  -0.865,   1.663,   0.123], [   0.513,   0.556,   1.800] ] ),
+                ( 160, [ [  -3.714,  -4.327,  10.954], [   0.547,   0.649,   0.376], [  -0.823,   1.989,   0.120], [  -0.146,  -0.164,   1.406] ] ),
+                ( 161, [ [  -3.160,  -3.622,  11.349], [   0.561,   0.761,   0.413], [  -0.742,   1.914,   0.073], [  -0.689,  -0.743,   1.012] ] ),
+                ( 162, [ [  -4.915,  -3.172,  10.679], [   0.583,   1.471,   0.333], [  -0.459,   1.818,   0.043], [   0.517,   0.369,   1.417] ] ),
+                ( 163, [ [  -4.169,  -1.970,  11.098], [   0.893,   0.891,   0.494], [  -0.086,   2.673,   0.165], [  -0.119,  -0.445,   1.054] ] ),
+                ( 164, [ [  -3.229,  -1.408,  11.614], [   0.940,   0.221,   0.512], [   0.569,   2.372,   0.432], [  -0.951,  -1.060,   0.793] ] ),
+                ( 165, [ [  -0.217, -10.725,  13.590], [   1.156,   1.585,  -0.231], [  -0.981,  -0.119,  -0.492], [  -2.549,   0.331,   2.963] ] ),
+                ( 166, [ [   0.880,  -9.221,  13.371], [   1.038,   1.423,  -0.207], [  -1.371,   0.432,   0.238], [  -0.069,  -1.365,   2.169] ] ),
+                ( 167, [ [  -0.930, -10.812,  13.232], [   0.439,   2.228,   0.148], [  -0.445,  -0.054,  -0.223], [  -1.164,   0.599,   3.690] ] ),
+                ( 168, [ [  -0.483,  -8.538,  13.383], [   0.456,   2.319,   0.154], [  -1.309,   0.920,  -0.222], [   0.113,  -0.514,   2.647] ] ),
+                ( 169, [ [  -1.652,  -7.413,  12.916], [   0.631,   0.433,   0.198], [  -1.200,   1.283,  -0.382], [  -0.152,  -0.300,   2.103] ] ),
+                ( 170, [ [  -2.862,  -5.978,  12.637], [   0.712,   0.498,   0.266], [  -1.004,   1.614,  -0.193], [  -0.096,   0.072,   1.905] ] ),
+                ( 171, [ [  -3.631,  -4.227,  12.536], [   1.045,   0.898,   0.696], [  -0.605,   1.704,  -0.233], [   0.302,   0.353,   1.714] ] ),
+                ( 172, [ [  -4.077,  -2.599,  12.188], [   0.790,   0.790,   0.975], [  -0.284,   1.540,  -0.459], [   0.296,  -0.794,   1.101] ] )
```

Comment:
For future reference; these parameters have only ~4 significant figures which is on the low side as they are using the number format for the much larger human mesh. I would have kept the number format used by Zohreh. Don't change it now as these are what they are, but next time.